spec/game_spec.rb

```diff
@@ -0,0 +1,101 @@
+require './lib/board'
+require './lib/game'
+require './lib/players'
+require './lib/choices'
+
+describe 'Tic tac toe' do
+  describe Players do
+    let(:players) { Players.new('paul') }
+    context '#initialize' do
+      it 'should accept an arg' do
+        expect { initialize('paul') }.to_not raise_error
+      end
+
+      it 'should set the instance variables @first_player_icon and @second_player_icon' do
+        expect(players.instance_variable_get(:@first_player_icon)).to eq('X')
+        expect(players.instance_variable_get(:@second_player_icon)).to eq('O')
+      end
+    end
+
+    context '#name' do
+      it "returns the player's name" do
+        name = 'Paul'
+        player = Players.new(name)
+        expect(player.name).to eq 'Paul'
+      end
+    end
+  end
+
+  describe Board do
+    let(:player1) { Players.new('Paul') }
+    let(:player2) { Players.new('Zay') }
+    let(:board) { Board.new(player1, player2) }
+
+    describe '#initialize' do
+      it 'should accept player names as args' do
+        board
+      end
```

Comment:
- [ ] This serves no purpose please remove this `it` block, as there is no `expect` case in it.